### PR TITLE
Millistone 1: Responsive Web Design

### PR DESCRIPTION
**Pull request by AbdumurodovaZulfizar**
<p>This project is for improving skills at Responsive Web Design.😊</p>
<h3>The Next Web Clone</h3>
<h5>Used Languages</h5>
<h4>HTML</h4>
<h5>Used tags</h5>
<ol>
    <li>main</li>
    <li>img</li>
    <li>div</li>
    <li>body</li>
    <li>a</li>
    <li>span</li>
    <li>footer</li>
    <li>body</li>
    <li>header</li>
    <li>head</li>
    <li>...</li>
</ol>
<h4>CSS</h4>
<h5>Used Properties</h5>
<ol>
    <li>linear-gradient background</li>
    <li>flex</li>
    <li>grid</li>
    <li>position:fixed</li>
    <li>media queries</li>
    <li>hidden visibility</li>
    <li>...</li>
</ol>

### CODE REVIEW

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS : CHANGES REQUIRED** :recycle: :recycle: 

@AbdumurodovaZulfizar, **you have done an extremely awesome job. I must commend your effort and your keen attention to details. Good job!!! :+1:  I also like your good use of flex through out the project.**

**This is an exceptionally complicated project so do not be tired or weary. We have some mandatory requirements you need to fix to get this project approved.**

### **GENERAL**
- [ ] Elements should be aligned with flex or grid in the footer. In your footer, other than the "footer-page" class which is the very last element in the page, you used other means to align your footer elements. Please make use of flex or grid in your footer. i.e the social media icons and the other menu options.
- [ ] You did not use any nav tag in the header. Use the nav tag for your menus.
- [ ] Use links (a tag) for all the menu options, the hamburger button and the search button. Currently, they are not.

**Top 3 articles section** 

- [ ] The title of each article is a header (hx tag) and also a link (a tag). Please make use of h tag.

**Latest Funding section**

- [ ] The title of the section is a header (hx tag) and also a link (a tag). Please use h tag.
- [ ] The nav menu (See more, Get updates, etc.) should be grouped in a nav tag.
- [ ] Each article in that section should have its own title (hx tag) and has a link (a tag). You currently have no title.
- [ ] The text excerpt at the bottom should also be a link (a tag).
- [ ] The nav menu (add info, get updates, see more) in desktop version(should be on the right of the header) and tablet version (should be below the header) should be in a row and separated by | . Yours is currently a column.

**Footer**

- [ ] Social media icons should be centered in a nav tag, each of them with a link (a tag).

**Other house keeping issues**
Below are some other design related adjustments.
- [ ] Make sure you don't have any unused classes. e.g class "conateiner1" is not in use anywhere in your stylesheet.
- [ ] Title article in the highlights section (the biggest one) is overflowing out of its container in tablet version, around 800px and below. This may be because the height of the images is visibly shorter in your version than the original.


[OPTIONAL]

- [ ] Try to align your divs properly to make your code easy to read by others and possibly, yourself in the future.

Please take care of these changes and submit another CRR. 

As described in the [Code reviews limit Policy ](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have 2 code reviews left for this project.

Please don't be tired, you are almost there. 

You can reach me at @george gbenle on slack if you require further clarification.
Happy coding. :smile:

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
-# Next-web-repo
\ No newline at end of file
+# The Next Web Colne
```
Comment (christianotieno) [inline README.md:1]: ## Improve the `README`
There is a typo in your header. Do you mind fixing this?
```suggestion
# The Next Web Clone
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                <div class="item1">
+                    <h3><a class="highlight-title" href="#">This Mac app automatically detects your duplicate files</a></h3>
+                    <p class="article-author">by <a href="#" class="author">Ivan Mehta </a><i class="fa fa-clock-o" aria-hidden="true"></i> 2 hours ago <i class="fa fa-share-alt" aria-hidden="true"></i> 5</p>
+                </div></div>
+            </div>
+        </div>
+
+
+        <div class="container3">
+            <h2><a class="container-titles" href="#">Latest funding rounds on Index.co  > </a></h2>
```
Comment (christianotieno) [inline index.html:73]: ## Improve the `Latest Funding` Section
- [ ] While conducting the above requested changes, you might have intoduece a bug that deformed the way the elements in the `Latest Funding`  section are aligned. We have some overlapping texts that need our attention. Do you mind fixing this section? See the comparison below 👇🏾 

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/AbdumurodovaZulfizar/Next-web-repo/millistone-1/index.html)|
|:--:|:--:|
|source|clone|

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @AbdumurodovaZulfizar 👋🏾 
Thank you for working on the majority of the changes. keep it up 👏🏾  
Below there is a suggestion on the `Latest funding` area that might need our attention. have a look 👇🏾 

> **"As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have only 1 code review left for this project. If you think that the code review was not fair, you can [request a second opinion using this form. ](https://airtable.com/shrQAqnBwek5a0O0s)"**

### Happy coding 💻 
#### Christian Otieno  (CR)

**Comment by AbdumurodovaZulfizar** [comment]
I have some questions:
1. Should I leave a new line at the end of my Readme.md file?
2.Should I change smth with my HTML file related to highlights? and Latest fundings?
Thank you for your help. Your review has been very useful.

**Comment by Berabjesus** [comment]
> I have some questions:
> 
> 1. Should I leave a new line at the end of my Readme.md file?
>    2.Should I change smth with my HTML file related to highlights? and Latest fundings?
>    Thank you for your help. Your review has been very useful.

Your readme file looks fine.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        <div class="footer-icons">
+          <nav>
+            <a class="icon" href="#"><i class="fa fa-facebook-official" aria-hidden="true"></i></a>
+            <a class="icon" href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            <a class="icon" href="#"><i class="fa fa-youtube-play" aria-hidden="true"></i></a>
+            <a class="icon" href="#"><i class="fa fa-instagram" aria-hidden="true"></i></a>
+            <a class="icon" href="#"><i class="fa fa-pinterest-p" aria-hidden="true"></i></a>            
+            <a class="icon" href="#"><i class="fa fa-wifi" aria-hidden="true"></i></a>
+            <a class="icon" href="#"><i class="fa fa-envelope" aria-hidden="true"></i></a>
+        </nav>
```
Comment (Berabjesus) [inline index.html:158]: ## Optional

- [ ] The footer icons here are not styled correctly. You may eventually have to work on it on the next milestone so it's best to do it now. 

#### Original 👇 
image

#### yours 👇 
image

**Review by Berabjesus — APPROVED** [review]
# **Status: PROJECT APPROVED** 
#### Excellent job on this project @AbdumurodovaZulfizar. 💯  Keep it up.  :champagne: :champagne: **All the required changes by the previous reviewer have been fulfilled. I will go ahead and mark this as complete. 👍** 

#### **Before you merge to the default branch please have a look at the following OPTIONAL comments below 👇**

I wish you all the very best in your next tasks:1st_place_medal:
#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
**HAPPY CODING :computer:[Bereket Beshane](https://bereketababu.me)** | [LinkedIn](https://linkedin.com/in/bereketbeshane) | [GitHub](https://github.com/Berabjesus)

**Pull request merged**